### PR TITLE
Updated kafka docker compose version + added Kafka Manager UI

### PR DESCRIPTION
**Pull request by pancudaniel7**

### CODE REVIEW

**Review by GeorgeH23 — APPROVED** [review]

File: deployments/docker/compose/kafka.yaml [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
-  kafka:
-    image: wurstmeister/kafka:2.12-2.5.0
-    hostname: kafka
-    ports:
-      - "9092:9092"
-    environment:
-      KAFKA_ADVERTISED_HOST_NAME: 127.0.0.1
-      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+zookeeper:
+  image: confluent/zookeeper
```
Comment (CodrinE) [inline deployments/docker/compose/kafka.yaml:4]: This image is like 6 years old, shouldn't we use something newer?

File: deployments/docker/compose/kafka.yaml [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
+    - KAFKA_DELETE_TOPIC_ENABLE=true
+    - KAFKA_LOG_RETENTION_HOURS=1
+    - KAFKA_MESSAGE_MAX_BYTES=10000000
+    - KAFKA_REPLICA_FETCH_MAX_BYTES=10000000
+    - KAFKA_GROUP_MAX_SESSION_TIMEOUT_MS=60000
+    - KAFKA_NUM_PARTITIONS=2
+    - KAFKA_DELETE_RETENTION_MS=1000
+
+kafka-manager:
+  image: sheepkiller/kafka-manager:latest
```
Comment (CodrinE) [inline deployments/docker/compose/kafka.yaml:26]: Alpine version should use less memory. Also "latest" tag name is not recommended.

File: deployments/docker/compose/kafka.yaml [hunk 3]
```diff
@@ -1,11 +1,35 @@
...
-    environment:
-      KAFKA_ADVERTISED_HOST_NAME: 127.0.0.1
-      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+zookeeper:
+  image: confluent/zookeeper
+  ports:
+    - "2181:2181"
+
+kafka:
+  image: wurstmeister/kafka:1.1.0
```
Comment (CodrinE) [inline deployments/docker/compose/kafka.yaml:9]: Can't we use a newer version?

Comment (pancudaniel7) [reply deployments/docker/compose/kafka.yaml:9]: This is the last version

File: deployments/docker/compose/kafka.yaml (hunk 2 above)
Comment (pancudaniel7) [reply deployments/docker/compose/kafka.yaml:26]: Very good point!

File: deployments/docker/compose/kafka.yaml (hunk 1 above)
Comment (pancudaniel7) [reply deployments/docker/compose/kafka.yaml:4]: I will add the version and check if I cand get a new image

File: deployments/docker/compose/kafka.yaml (hunk 3 above)
Comment (CodrinE) [reply deployments/docker/compose/kafka.yaml:9]: Latest version is 2.13-2.8.1 if I am not wrong.

**Pull request merged**